addons/mail/models/res_partner.py

```diff
@@ -20,6 +22,15 @@ class Partner(models.Model):
     vat = fields.Char(tracking=5)
     # channels
     channel_ids = fields.Many2many('mail.channel', 'mail_channel_member', 'partner_id', 'channel_id', string='Channels', copy=False)
+    # add tracked field
+    contact_address_inline = fields.Char(compute='_compute_contact_address_inline', string='Inlined Complete Address', tracking=True)
+
+    @api.depends('contact_address')
+    def _compute_contact_address_inline(self):
+        """Compute an inline-friendly address, not guaranteed to be a valid postal format."""
```

Comment:
Not sure why we bother to do that, specifically because otherwise the tracking looks bad? If so, would be nice to explain.

Tracking the origin field would be much simpler but I'm guessing there is a reason.

Reply:
It's because tracking messages use <li> nodes with inline text. If we put some '\n' in there it will just ignore it. And the address ends up difficult to read as there's then no distinction between the different elements of the address.

I added a comment above the field itself to justify its existence.